MIDAS/lib/TCAL9539/TCAL9539.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include<Arduino.h>
+
+struct GpioAddress {
```

Comment:
I feel like this should be named GPIOAddress

Reply:
I think Google convention is what Nick wrote